pkgs/development/python-modules/flask-migrate/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k, python, glibcLocales, flask, flask_sqlalchemy, flask_script, alembic
+}:
+
+with stdenv.lib;
+
+buildPythonPackage rec {
+  pname = "Flask-Migrate";
+  version = "2.0.3";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "107x78lkqsnbg92dld3dkagg07jvchp3ib3y0sivc4ipz6n1y7rk";
+  };
+
+  # When tests run with python3*, tests should run commands as "python3 <args>",
+  # not "python <args>"
+  patchPhase = ''
+    substituteInPlace tests/test_migrate.py --replace "python" "${python.executable}"
```

Comment:
there's a symlink from `python` to `python3` so it should work fine without this fix